src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 20px;
+  background-color: #000;
+}
+
+.external {
+  box-sizing: border-box;
+  width: 350px;
+  background-color: #2c8ec2;
+  border: 1px solid #dc5757;
+  margin-bottom: 40px;
+}
+
+.middle {
+  box-sizing: border-box;
+  background-color: #ffa600;
+  border: 1px solid #dc5757;
+  margin: 10px;
```

Comment:
border and margin used in few places
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.